.github/workflows/build-lint-test.yml

```diff
@@ -89,6 +89,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'yarn'
       - run: yarn --immutable --immutable-cache
+      - run: yarn build
       - run: yarn test
```

Comment:
Is this correct?

Reply:
I believe so because running `yarn build` from the root or snap will build the snap bundle
<img width="795" alt="Screenshot 2024-03-29 at 11 52 13 AM" src="https://github.com/MetaMask/snap-watch-only/assets/91970214/9fff0dac-0ac4-45b2-baa7-523fb8d7ff27">